docs/publish/host-an-office-add-in-on-microsoft-azure.md

```diff
@@ -1,225 +1,180 @@
     
-4. Choose  **File**, and then choose  **Save All**. Close the add-in manifest properties page.
+3. In the XML manifest file, find and replace all instances of "~remoteAppUrl" with the root URL of the add-in web app on Azure. This is the URL that you copied earlier after you published the add-in web app to Azure (for example: https://YourDomain.azurewebsites.net). 
     
-5. Back in  **Solution Explorer**, right-click the manifest folder and choose  **Open Folder In File Explorer**.
+4. Choose **File** and then choose **Save All**. Close the add-in XML manifest file.
     
-6. Copy the add-in manifest file, for example OfficeAdd-in1.xml. 
+5. Back in **Solution Explorer**, right-click the manifest folder and choose **Open Folder In File Explorer**.
     
-7. Browse to the network file share that you created earlier in the topic and paste the manifest file into the folder.
+6. Copy the add-in XML manifest file (for example, WordWebAddIn.xml). 
     
+7. Browse to the network file share that you created in [Step 1: Create a shared folder](../publish/host-an-office-add-in-on-microsoft-azure.md#step-1-create-a-shared-folder-to-host-your-add-in-xml-manifest-file) and paste the manifest file into the folder.
 
 ## Step 7: Insert and run the add-in in the Office client application
 
-
-
-1. Start Word and open a new document.
+1. Start Word 2016 and create a document.
     
-2. On the Ribbon, choose  **Insert** > **My Apps**, and then choose  **See all**.
+2. On the ribbon, choose **Insert** > **My Add-ins**. 
     
-3. In the  **Apps for Office** dialog box, choose **SHARED FOLDER**. Office client applications that work with the Office Add-ins model scan the folder that you list as a trusted add-in catalog and show the add-ins in the dialog. You should see the icon for your sample add-in.
+3. In the **Office Add-ins** dialog box, choose **SHARED FOLDER**. Word scans the folder that you listed as a trusted add-ins catalog (in [Step 2: Add the file share to the Trusted Add-ins catalog](../publish/host-an-office-add-in-on-microsoft-azure.md#step-2-add-the-file-share-to-the-trusted-add-ins-catalog)) and shows the add-ins in the dialog. You should see an icon for your sample add-in.
     
-4. Choose the icon for your add-in and then choose  **Insert**. The add-in is inserted on the side of the client application.
-    
-5. Test that the add-in is working by creating some text in the document, then selecting the text, and then choosing  **Get data from selection**.
+4. Choose the icon for your add-in and then choose **Add**. A **Show Taskpane** button for your add-in is added to the ribbon. 
+
+5. On the ribbon of the **Home** tab, choose the **Show Taskpane** button. The add-in opens in a task pane to the right of the current document.
     
+6. Test that the add-in works by selecting some text in the document and choosing the **Highlight!** button in the task pane. 
```

Comment:
Change to either "Verify that the add-in works", or "To test the add-in, select... and choose... ".